### PR TITLE
test: fix wrong result introduced by onerror

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "test": "npm run lint -- --fix && npm run test-local",
     "test-local": "egg-bin test",
     "cov": "egg-bin cov",
-    "ci": "npm run lint && npm run cov"
+    "ci": "egg-bin pkgfiles && npm run lint && npm run cov",
+    "pkgfiles": "egg-bin pkgfiles"
```
Comment (fengmk2) [inline package.json]: 看来都得加上

Comment (atian25) [inline package.json]: 看来骨架要改改

Comment (atian25) [inline package.json]: @popomore 应该用 `egg-bin pkgfiles --check` 吧，不报错的话，放在 ci 也发现不了。 或者放在 lint

Comment (atian25) [inline package.json]: https://github.com/eggjs/egg-boilerplate-plugin/pull/19

**Pull request merged**